libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/persistence/Utils.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.openhouse.datalayout.persistence;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import com.linkedin.openhouse.datalayout.layoutselection.DataLayoutOptimizationStrategy;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.spark.sql.SparkSession;
+
+/** Utility class for strategies persistence. */
+public final class Utils {
+  public static final String DATA_LAYOUT_PROPERTY_KEY = "data-layout-strategies";
+
+  private Utils() {}
+
+  public static void saveStrategies(
```

Comment:
Are we going to store DLO strategy and config in the corresponding table properties? It seems it is part of table properties. Or is there a plan to maintain them separately?

Reply:
Yes, the intention is to keep strategies in a table property